library/errata_tool_product_version.py

```diff
@@ -42,6 +42,26 @@
                redhatrelease2, redhatengsystems]
      required: false
      default: redhatrelease2
+   use_quay_for_containers:
+     description:
+       - If enabled, the Errata Tool will instruct pub to push this product
+         version's container advisories to quay.io instead of docker-pulp for
+         live pushes.
+       - Leave this unchecked unless your sure this product version
+         is ready for Quay.io.
```

Comment:
`your` -> `you're`, but let's just simplify this...
```suggestion
       - Leave this unchecked unless this product version is ready
         for Quay.io.
```